go/stats/opentsdb/opentsdb.go

```diff
@@ -43,7 +43,9 @@ func sendDataPoints(data []dataPoint) error {
 	if err != nil {
 		return err
 	}
-	resp.Body.Close()
+	if err := resp.Body.Close(); err != nil {
```

Comment:
Go styling guide recommends `return resp.Body.Close()` instead. Same comment below.

Reply:
Thank you for you suggestion.
I changed it.